### PR TITLE
hardening/1182_deprecate_gr_pending_doc_fixes

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes pending documentation issues of #1182

### CODE REVIEW

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #<a name="top"></a>NGSIGroupingInterceptor
+**IMPORTANT NOTE: from release 1.6.0, this feature is deprecated in favour of Name Mappings.**
```
Comment (fgalan) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md]: A link to the deprecated_and_removed.md file, to the place were this deprecation is explained, could help.

File: doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md [hunk 2]
```diff
@@ -30,7 +30,12 @@ Deprecated features are removed not before 3 developement sprints (usually, a de
 [Top](#top)
 
 ##<a name="section2"></a>Deprecated functionalities
-Currently, there is no deprecated functionality.
+###<a nanem="section2.1"></a>Grouping Rules
+Added after releasing version [0.5](https://github.com/telefonicaid/fiware-cygnus/releases/tag/release-0.5) (issue [107](https://github.com/telefonicaid/fiware-cygnus/issues/107)).
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md]: "Added after releasing version 0.5" vs "Added at version 0.5". Maybe the later is what you really want to say?

File: doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md [hunk 3]
```diff
@@ -30,7 +30,12 @@ Deprecated features are removed not before 3 developement sprints (usually, a de
...
+
+Deprecated after releasing version [1.6.0](https://github.com/telefonicaid/fiware-cygnus/releases/tag/1.6.0) (issue [1182](https://github.com/telefonicaid/fiware-cygnus/issues/1182)).
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md]: Similar comment here.

File: doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md [hunk 4]
```diff
@@ -1,4 +1,6 @@
 #<a name="top"></a>Grouping Rules
+**IMPORTANT NOTE: from release 1.6.0, this feature is deprecated in favour of Name Mappings.**
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md]: A link to the deprecated_and_removed.md file, to the place were this deprecation is explained, could help.

File: doc/cygnus-ngsi/installation_and_administration_guide/name_mappings.md [hunk 5]
```diff
@@ -183,7 +183,7 @@ As seen, the Name Mappings feature is quite similar to the already existent grou
 |Plain Flume `Event`'s are intercepted, and `NGSIEvent`'s are put into the channel. Because the interceptor needs to parse the original notification, a `NGSIEvent` already contains the original notification parsed, and the mapped version of the original notification, <b>freeing the sinks to parse the notification</b>.|Plain Flume `Event`'s are intercepted, and plain `Event`'s are put into the channel. Thus, <b>the sinks must parse the notification</b>, despite the grouping interceptor already parsed it`.|
 |It is expected a `enable_content_mappings` feature is implemented in the future. Such a content mapping will take advantage of the already mapped version on the original notification within `NGSIEvent`'s.|Such a functionality is very hard to implement based on the current grouping interceptor code.|
 
-Most probably, the grouping rules feature is deprecated in future versions of Cygnus.
+**IMPORTANT NOTE: from release 1.6.0, Grouping Rules are deprecated in favour of Name Mappings.**
```
Comment (fgalan) [inline doc/cygnus-ngsi/installation_and_administration_guide/name_mappings.md]: A link to the deprecated_and_removed.md file, to the place were this deprecation is explained, could help.

File: doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md (hunk 1 above)
Comment (frbattid) [inline doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_grouping_interceptor.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1312/commits/1012fdb5693a7ed0cbd3164d46d5f10c53de5baf

File: doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md (hunk 2 above)
Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1312/commits/1012fdb5693a7ed0cbd3164d46d5f10c53de5baf

File: doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md (hunk 3 above)
Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/deprecated_and_removed.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1312/commits/1012fdb5693a7ed0cbd3164d46d5f10c53de5baf

File: doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md (hunk 4 above)
Comment (frbattid) [inline doc/cygnus-ngsi/installation_and_administration_guide/grouping_rules.md]: Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1312/commits/1012fdb5693a7ed0cbd3164d46d5f10c53de5baf

**Comment by fgalan** [comment]
LGTM

**Pull request merged**